assembl/static2/js/app/components/home/objectives.jsx

```diff
@@ -33,7 +33,8 @@ const Objectives = ({ debate, timeline, lang, buttonLabel }: Props) => {
     locale = 'zh_CN';
   }
 
-  const haveImages = objectives.images.img1Url && objectives.images.img2Url;
+  const haveOneImage = objectives.images.img1Url !== '' || objectives.images.img2Url !== '';
```

Comment:
Ahh, a bit of tech debt built up here because of 8de691d39fa7ee78ce240272757354ce7a745174 branch. It could be alright, we can refactor when that branch is ready by converting your changes into the various configurations.

Reply:
yeah you noticed that also...